src/test/java/br/ufg/inf/AprovDivulgInfoTest.java

```diff
@@ -0,0 +1,482 @@
+package br.ufg.inf;
+
+import static org.junit.Assert.*;
+
+import java.io.File;
+import java.io.FileInputStream;
+import java.io.FileNotFoundException;
+import java.io.UnsupportedEncodingException;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.sql.Connection;
+import java.sql.DatabaseMetaData;
+import java.sql.Date;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+
+public class AprovDivulgInfoTest {
+
+	// conex�o com db
+	static String url = "jdbc:postgresql://localhost:5432/AprovDivulgInfoDB";
+	static String usuarioDB = "postgres";
+	static String senhaDB = "admin";
+
+	// usuario
+	static String email = "email";
+	static String senha = "senha";
+	static String nome = "nome";
+	static String cpf = "00000000001";
+	static String recebe_divulgacao = "DIARIA";
+	static Date timestamp_de_cadastramento = new Date(0000, 00, 00);
+	static Date timestamp_de_ultima_atualizacao = new Date(0000, 00, 00);
+	static Date timestamp_de_exclusao_logica = new Date(0000, 00, 00);
+
+	// aprovacao_de_divulgacao
+	static int id_aprovacao = -1;
+	static boolean aprovada = false;
+	static String parecer = "parecer sobre divulgacao";
+	static Date data_aprovacao_ou_rejeicao = new Date(0000, 00, 00);
+
+	// curso_da_ufg
+	static String nivel = "MESTRADO";
+	static String tipo_de_resolucao = "CEPEC";
+	static int numero_da_resolucao = 2;
+	static boolean presencial = false;
+	static String turno = "INTEGRAL";
+
+	@Test
+	public void testDriverPostgresql1() throws SQLException, ClassNotFoundException {
+
+		Class.forName("org.postgresql.Driver");
+
+	}
+
+	@Test
+	public void testConnectionDB() throws SQLException {
+
+		Connection conn = DriverManager.getConnection(url, usuarioDB, senhaDB);
+
+	}
```

Comment:
Estes testes de conexão não deveriam estar aqui. O arquivo de testes deveria testar apenas a funcionalidade que ele deveria. Entendo para verificar se a tabela existe, mas estes seriam mais "globais", e deveriam antes de todos os outros testes.
Como uma forma de fazer isso: https://github.com/junit-team/junit4/wiki/aggregating-tests-in-suites